certification/internal/engine/crane.go

```diff
@@ -11,6 +14,7 @@ func NewCraneEngine() *craneEngine {
 func (c *craneEngine) ListTags(imageURI string) ([]string, error) {
 	// prepare crane runtime options
 	options := make([]crane.Option, 0)
```

Comment:
Small nit, but would be more efficient to give it a length 0, capacity of 1.

Reply:
Yeah, I can change that. Was more focused on trying to test this.